_posts/2019-05-06-live-view-with-presence.md

```diff
@@ -0,0 +1,582 @@
+---
+author: Sophie DeBenedetto
+author_link: https://github.com/sophiedebenedetto
+categories: post
+d©ate: 2019-05-06
```

Comment:
We need to update the date and looks like you've got a copyright character in there.

Reply:
Good catch! fixed and updated to today's date.